pkg/telemetry/phonehome/utils.go

```diff
@@ -0,0 +1,23 @@
+package phonehome
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
```

Comment:
nit: doesn't seem to be used within this file but rather `client_config.go / gatherer.go`, so might make sense to move it closer to where it's actually used.